muspinsim_input/muspinsim_input.xml

```diff
@@ -1,4 +1,5 @@
-<tool id="muspinsim_input" name="Create Muspinsim input file" version="@TOOL_VERSION@+galaxy@WRAPPER_VERSION@" python_template_version="3.5" profile="22.01">
+<tool id="muspinsim_input" name="Create Muspinsim input" version="@TOOL_VERSION@+galaxy@WRAPPER_VERSION@" python_template_version="3.5" profile="22.01">
```

Comment:
Is it worth changing this id to `muspinsim_config` to match `pm_yaml_config`? (would need to be updated in folder/file name, shed.yml, etc as well)